autoscale_cloudcafe/autoscale/behaviors.py

```diff
@@ -480,13 +480,11 @@ def retry(self, callable, timeout=None, interval_time=None,
         if interval_time is None:
             interval_time = self.autoscale_config.interval_time
 
-        if time_scale:
+        if time_scale and self.autoscale_config.mimic:
             # scale time down if using mimic - no shorter than 1 second, though
```

Comment:
Since we are no longer doing `max(val, 1)`, the last half of this comment can probably be truncated.  :)